postgresql/postgresql.go

```diff
@@ -237,14 +244,58 @@ func interfaceToString(face interface{}) (string, error) {
 			ret += ", "
 			ret += strconv.Itoa(val[i])
 		}
-	case int:
-		ret = strconv.Itoa(val)
+	case int64:
+		ret = strconv.FormatInt(val, 10)
+
+	case []int64:
+		length := len(val)
+		if length == 0 {
+			return ret, err
+		}
+		ret = strconv.FormatInt(val[0], 10)
+		if length == 1 {
+			return ret, err
+		}
+		for i := 1; i < length; i++ {
+			ret += ", "
+			ret += strconv.FormatInt(val[i], 10)
+		}
+	case uint64:
+		ret = strconv.FormatUint(val, 10)
+
+	case []uint64:
+		length := len(val)
+		if length == 0 {
+			return ret, err
+		}
+		ret = strconv.FormatUint(val[0], 10)
+		if length == 1 {
+			return ret, err
+		}
+		for i := 1; i < length; i++ {
+			ret += ", "
+			ret += strconv.FormatUint(val[i], 10)
+		}
+
 	case float64:
 		ret = strconv.FormatFloat(val, 'g', -1, 64)
-	case string:
-		ret = val
+
+	case []float64:
+		length := len(val)
+		if length == 0 {
+			return ret, err
+		}
+		ret = strconv.FormatFloat(val[0], 'g', -1, 64)
+		if length == 1 {
+			return ret, err
+		}
+		for i := 1; i < length; i++ {
+			ret += ", "
+			ret += strconv.FormatFloat(val[i], 'g', -1, 64)
+		}
+
 	default:
-		err = errors.New("unsupported type")
+		err = fmt.Errorf("Unsupported type %v (currently supported data type: string, []string, bool, int, []int, int64, uint64, []uint64, float64, []float64)", reflect.TypeOf(val))
```

Comment:
Would it be worth adding `uint` and `[]int64`  so that the support is consistent between int/unit?